sylvia-derive/src/parser.rs

```diff
@@ -191,6 +195,7 @@ impl MsgAttr {
         let mut name = Ident::new("InstantiateMsg", content.span());
         let p: Option<Token![,]> = content.parse()?;
 
+        // In what scenario is this true?
```

Comment:
Not a valid comment. It may be true if some args (I think the name in particular) are passed to instantiate. However I don't know it this makes sense, for now, possibly this chunk could be removed).